### ISSUE TITLE
Small remark on --cv-ratio (i.e. validation set)

### ISSUE THREAD

**Issue by nd26** [ildoonet/pytorch-randaugment#16]
Hey,

I am not sure if --cv-ratio was intended for use but I believe the wrong set of transformations are given to the respective dataloader, i.e. the validation dataloader is given the transformations of the training set (including RandAugment transformations).
